crates/bevy_ecs/src/system/commands/mod.rs

```diff
@@ -956,6 +949,15 @@ where
     }
 }
 
+impl<T> Remove<T> {
+    fn new(entity: Entity) -> Self {
```

Comment:
We're trying to eventually document every public API in this crate. Can you fill in a short doc comment on this and the other constructors?